### DISCUSSION TITLE
Stats shown after enter press key

### DISCUSSION THREAD

**Discussion by pdelteil** [projectdiscovery/nuclei#1269] [Q&A] [answered]
**Describe the bug**
Not really sure if it's an issue (maybe a question). While I run the scan in file mode (not pipe mode), when I press enter I get the stats of the scan. Using pipe mode (something like ``somecommand | nuclei -t templates -stats -si 120``) I get the stats after -si time nothing happens after pressing the enter key. 

**Nuclei version**
 2.3.4

**Comment by ehsandeep** [comment, marked as answer]
@pdelteil Yes, you are right, with file input, we can see the stats upon key press (initial behavior of this support), maybe need some cleanups and could be also useful for some.
